src/OpenSEE/Scripts/TSX/defaults.ts

```diff
@@ -244,12 +244,12 @@ export const defaultSettings = {
     } as OpenSee.IUnitCollection,
     TimeUnit: {
         current: 5, options: [
```

Comment:
you may want to change `current` to `0` since it was `auto`

Reply:
That's right. If you don't change this, the default unit for time will be `seconds` instead of `auto`.

Reply:
Ok, went ahead and fixed it